operator/gce.go

```diff
@@ -95,6 +98,10 @@ func (r *GCEShutdownCall) ShutdownWithInterval(ctx context.Context, interval tim
 		res = multierror.Append(res, err)
 	}
 
+	if valuesGCE(r.TargetList.Items) == nil {
+		return nil, nil
+	}
+
```

Comment:
🔧I thing this needsto be changed.
This code should be unnecessary because this is illegal that map is nil.
Please investigate the cause of map becoming nil.